mmv1/third_party/terraform/resources/resource_compute_region_instance_template.go.erb

```diff
@@ -0,0 +1,1182 @@
+<% autogen_exception -%>
+
+package google
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/customdiff"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+<% if version == "ga" -%>
+	"google.golang.org/api/compute/v1"
+<% else -%>
+	compute "google.golang.org/api/compute/v0.beta"
+<% end -%>
+)
+
+func ResourceComputeRegionInstanceTemplate() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceComputeRegionInstanceTemplateCreate,
```

Comment:
It's confusing to me why this schema is so different from `resource_compute_instance_from_template`. Should we not follow the pattern and mirror the schema of Compute Instance to define this otherwise this resource will drift from Compute Instance. 

In my mind these resources are of the same sort.

Reply:
According to timus@, there are more fields which are supported in Instance in TF but not supported in InstanceTemplate. he has an analysis in this [spreadsheet](https://docs.google.com/spreadsheets/d/1Yu1ted0VZ0Clpbhls-3mZ9HMSBAnY9bVsIx60nrvd-A/edit#gid=157434453). Instances team has a plan to fix it but it was postponed to May.

The decision is we shouldn't block RIT launch if all fields supported currently in GIT are also supported in RIT.